.travis.yml

```diff
@@ -23,35 +24,58 @@ env:
    - IOS_SDK=iphonesimulator12.1
    - TVOS_SDK=appletvsimulator12.1
 
-   matrix:
-   - DESTINATION="OS=11.4,name=iPhone X" SDK="$IOS_SDK" BUILD_EXAMPLE="YES"
-
-   - DESTINATION="OS=12.1,name=iPhone X" SDK="$IOS_SDK" BUILD_EXAMPLE="YES"
-
-   - DESTINATION="OS=12.1,name=Apple TV" SDK="$TVOS_SDK" BUILD_EXAMPLE="NO"
-
 before_install:
   - pip3 install proselint
   - gem update --system
   - gem install bundler
 
-script:
-- set -o pipefail
-- xcodebuild -version -sdk
+jobs:
+  include:
+    - &test
+      stage: test
+      name: iOS iPhone X 11.4
```

Comment:
can we just use the OS name? 

so: `iOS 11.4`, `iOS 12.1`, `tvOS 12.1`

Reply:
Sounds good.  Ill update those now.